src/forklift/core.py

```diff
@@ -155,26 +187,83 @@ def _move_data(crate):
             source = temp_table
 
     sql_clause = None
-    if 'OBJECTID' in [f.name for f in arcpy.ListFields(crate.source)] and 'OBJECTID' in [f.name for f in arcpy.ListFields(crate.destination)]:
-        sql_clause = (None, 'ORDER BY OBJECTID')
+    objectIdField = 'OBJECTID'  # arcpy.Describe(source).OIDFieldName
+    if objectIdField in [f.name for f in arcpy.ListFields(crate.source)]:
+        sql_clause = (None, 'ORDER BY {}'.format(objectIdField))
 
     try:
-        with arcpy.da.InsertCursor(crate.destination, fields) as icursor, \
-                arcpy.da.SearchCursor(source, fields, sql_clause=sql_clause) as cursor:
-            for row in cursor:
-                if shape_token not in fields or row[-1] is not None:
-                    icursor.insertRow(row)
-
-        edit_session.stopOperation()
-        edit_session.stopEditing(True)
-        log.debug('edit session stopped')
+        newRows = []
+        hashLookup, geoHashLookup = _get_hash_lookups(crate.destination)
+        orderNum = 0
+        atthashCount = 0
+        with arcpy.da.SearchCursor(source, fields, sql_clause=sql_clause) as srcCursor:
+            for row in srcCursor:
+                orderNum += 1
+                # Shape hash
+                geoHexDigest = None
+                if not is_table:
+                    wkt = row[-1].WKT
+                    geoHexDigest = _create_geo_hash(wkt, orderNum)
+                # Attribute hash
+                hasher = hashlib.md5()
```

Comment:
Yeah the only way to reset a hash with hashlib is to create a new instance. When I profiled this the instance creation didn't seem to take much time, which is nice.